src/main/java/com/bulkbuy/enterprise/controller/NavigationController.java

```diff
@@ -1,13 +1,15 @@
 package com.bulkbuy.enterprise.controller;
 import com.bulkbuy.enterprise.dto.Order;
 import com.bulkbuy.enterprise.service.IOrderService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
 @Controller
 public class NavigationController {
 
+    @Autowired
```

Comment:
Added the @Autowired annotation so the interface could be found by spring boot, otherwise it was a null variable.